flink-runtime/src/main/java/org/apache/flink/runtime/resourcemanager/ResourceManager.java

```diff
@@ -1070,6 +1074,14 @@ protected void setFailUnfulfillableRequest(boolean failUnfulfillableRequest) {
 		slotManager.setFailUnfulfillableRequest(failUnfulfillableRequest);
 	}
 
+	protected void startStartupPeriod() {
+		// noop
+	}
+
+	protected void resetStartupPeriod() {
```

Comment:
I think we don't need this method. Instead we could let the `startStartupPeriod` always call `setFailUnfillableRequests(false)`.